runtime/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -0,0 +1,59 @@
+package org.corfudb.runtime.view;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.view.ClusterStatusReport.NodeStatus;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ManagementViewTest {
+    private final LayoutUtil layoutUtil = new LayoutUtil();
+    private final ManagementView managementView = new ManagementView(Mockito.mock(CorfuRuntime.class));
+
+    /**
+     * Check that getting a quorum layout from the list of layouts works as expected
+     */
+    @Test
+    void testGetLayoutForQuorum() {
+        final String server1 = "server1";
+        final String server2 = "server2";
+        final String server3 = "server3";
+
+        final List<String> servers = Arrays.asList(server1, server2, server3);
+        final Layout layout = layoutUtil.getLayout(servers);
+
+        Map<String, Layout> layouts = new HashMap<>();
+
+        servers.forEach(server -> layouts.put(server, layout));
+
+        Optional<Layout> quorumLayout = managementView.getLayoutFromQuorum(layouts, layouts.size() - 1);
+        assertThat(quorumLayout).isEqualTo(Optional.of(layout));
+
+        quorumLayout = managementView.getLayoutFromQuorum(layouts, layouts.size());
+        assertThat(quorumLayout).isEqualTo(Optional.of(layout));
+
+        quorumLayout = managementView.getLayoutFromQuorum(layouts, layouts.size() + 1);
+        assertThat(quorumLayout).isEqualTo(Optional.empty());
+    }
+
+    @Test
```

Comment:
This test already exists in queryClusterStatus in this same file.

Reply:
The advantages of the unit test:
 - no side-effects (no need to start external services - nutty, no need for network communication, etc)
 - much faster
 - simpler to understand, simpler to find what causes the error.

I could delete those checks in `queryClusterStatus` method if you would like to. 

Also, I think it's ok to have integration and unit tests even if they check similar parameters.